Source/WebCore/html/PluginDocument.cpp

```diff
@@ -62,19 +66,44 @@ class PluginDocumentParser final : public RawDataDocumentParser {
 
     void appendBytes(DocumentWriter&, const uint8_t*, size_t) final;
     void createDocumentStructure();
+    Ref<HTMLStyleElement> createStyleElement(Document&);
 
     WeakPtr<HTMLEmbedElement, WeakPtrImplWithEventTargetData> m_embedElement;
 };
 
+Ref<HTMLStyleElement> PluginDocumentParser::createStyleElement(Document& document)
+{
+    auto styleElement = HTMLStyleElement::create(document);
+
+    constexpr auto styleSheetContents = R"CONTENTS(
+html { width: 100%; height: 100%; }
+body { width: 100%; height: 100%; margin: 0; overflow: hidden; }
+embed { width: 100%; height: 100%; }
+)CONTENTS";
```

Comment:
Can't we use `_s` suffix to make it an ASCIILiteral? It would avoid the call to `String::fromLatin1()` below.